### PR TITLE
New idea wondering what your opinions are?

### PR DESCRIPTION
**Pull request by itsmat32143**
@PoorPocketsMcNewHold @nickspaargaren 

I was looking at how GitHub displays the text file into a web page and copied the code 

Which allowed me to change the text file to a html file and use GitHub pages to host it giving it the link

[https://nickspaargaren.github.io/no-google/pihole-google.html](https://nickspaargaren.github.io/no-google/pihole-google.html)

This could allow for a custom domain to be given to the link or for the link to be a bit shorter if file names where changed it also allowed for me to add a head into the html file and include social media preview so if someone shared the link above on social media it would show the gafam picture and a title and description i also added into the head mobile view so if you open the link on a phone you do not have to zoom in because I use my phone most of the time to check the link to see if the date has changed so i know If I need to update my pi hole or not

I have only done the pi hole file as a test to see what uses think anyway what are your thoughts on this ???

### CODE REVIEW

**Comment by nickspaargaren** [comment]
Hi Matty,
I really appreciate your contribution, it looks like a great addition to the repo! The thing is that if we change te filename/structure of the file, the Pi-hole (or other tools) won't able to read the domains in the list. I believe this is the reason why @llacb47 decided to downvote your suggestion and why I have to close this issue.

**Comment by itsmat32143** [comment]
https://raw.githubusercontent.com/nickspaargaren/no-google/master/pihole-google.txt

@nickspaargaren 
@llacb47 

i thought the exact same as both of you at first if you open the code of the current link the one above and inspect the code and go to elements you will see its a html page with a head and body like a standard website the pihole only reads the body text and doesnt look at the head allowing you to hide stuff in the head like social preview features and mobile view and much more

https://itsmat32143.github.io/test/

As a proof of concept above is a GitHub page website with the google list in it but if you inspect the code of this list you will see that i have added social preview code and mobile view if you open the link in your phone or tablet it will fit the screen and you wont have to zoom in this has all been added into the head which pi hole doesn't read but the body text is exactly the same as the current list 

when running my url though pihole you get what i have attached below and you can test yourself 


  [i] Neutrino emissions detected...
  [✓] Pulling blocklist source list into range

  [✓] Preparing new gravity database
  [i] Target: https://itsmat32143.github.io/test/
  [✓] Status: Retrieval successful
  [i] Received 4785 domains

  [✓] Storing downloaded domains in new gravity database
  [✓] Building tree
  [✓] Swapping databases
  [i] Number of gravity domains: 4785 (4760 unique domains)
  [i] Number of exact blacklisted domains: 5
  [i] Number of regex blacklist filters: 1
  [i] Number of exact whitelisted domains: 9
  [i] Number of regex whitelist filters: 0
  [✓] Cleaning up stray matter

  [✓] DNS service is running
  [✓] Pi-hole blocking is Enabled


In conclusion basically instead of GitHub making the basic web page with the list on it you make it yourself and can hide whatever you want into the head like social preview and mobile view and i am sure many other things it also lets you make a custom link using a domain that you would add to the repo or you can just make the link shorter than the huge link you normally have to copy to pihole

**Comment by llacb47** [comment]
Serving your hosts file as HTML is just not a good practice. I don't think you have given any good reasons for it. Also, please punctuate your sentences.
